scripts/pyinstaller.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash -i
```

Comment:
Why interactive shell?

Reply:
It shouldn't be there, thanks for the catch